scipy/special/_support_alternative_backends.py

```diff
@@ -101,11 +100,13 @@ def __chdtr(v, x):
 
 
 def _chdtrc(xp, spx):
+    if spx is None:
+        return None
```

Comment:
This is boilerplate, so it should probably be moved elswehere though. I think we can leave it out if  line 35 becomes:
```python3
if f_name in _generic_implementations and spx is not None:
```